src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -708,5 +744,155 @@ public static nuint RotateRight(nuint value, int offset)
             return (nuint)RotateRight((uint)value, offset);
 #endif
         }
+
+        /// <summary>
+        /// Calculates the CRC (Cyclic redundancy check) checksum.
+        ///
+        /// Uses following Hardware Intrinsics, if supported:
+        /// * uint32_t __crc32b (uint32_t a, uint8_t b)
+        /// * uint32_t __crc32cb (uint32_t a, uint8_t b)
+        /// </summary>
+        /// <param name="crc">The base value to calculate checksum on</param>
+        /// <param name="data">The checksum data</param>
+        /// <returns>The CRC-checksum</returns>
+        [CLSCompliant(false)]
+        public static uint Crc32C(uint crc, byte data)
+        {
+            if (Sse42.IsSupported)
+            {
+                // uint32_t __crc32b (uint32_t a, uint8_t b)
+                return Sse42.Crc32(crc, data);
+            }
+            if (Crc32.IsSupported)
+            {
+                // uint32_t __crc32cb (uint32_t a, uint8_t b)
+                return Crc32.ComputeCrc32C(crc, data);
+            }
+
+
+            // Software fallback
```

Comment:
still doesn't produce the same result as Sse42.Crc32 with `Crc32(1164749927, byte.MaxValue)`

Reply:
I know 😦... Do you know why? [SharpLab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEUCuA7AHwAEAmABgFgAoUgRmqLIAIjaA6AJXwwEsBbGGwCSeDFB54AzjzCS2ADQAcSANzVqeAIYDJAB01gYTACoxJGJgG9qTWyxIBmK3ZeuXNty1pImOCRgBtAF0mMCgwY01gABsjAF4mPBgAd19/YI83aypPVzIEMjQmfIAxEiQAIUUHMgci/IBRWgcKgHYyEtb6hGaAVg6HEvRihABhVoBOAEEJ1toS7odektpaFFpR7vKp2gbWhsVugBEUUanUBoq0TNyRxSmjid7FUYWR1sUKkiOK0c2RyoNEgkaqHEYTCaKaYVFrdFAnByvI6FEYVMqKU5HOojKbjT5lLojXpNByIkjoG65fK0MjE8ZIN6NRy9FAoJD/UreXptSGLMgNBwoCZXEZHVqjFD9dpbXpTEpY1rY/IOBllMiKQn5CUTFBTBwY645W62fLTDUTEpTFH5JCKBqjDZ67qtCq/BolXqahBQsXslndXpHWjI/HdOWtW2ujkIX7lD4oMH5FANeElCooK6Uzz5cgVI52o4NY7fJBkI7l7oO5QlIETRYOBpio4kKbdPYkOYTB3dWgs0bTUYkbpIDZICYq7pDVr7PW9Q3Gk0IKbneEj2gBxQdXq9WhDkYZ5bnBpIboVccoEjEucAo419b87oWp6DsvdF6jXoOeZ194NKaXEhLXnBdlX5QVhXXEZB0UF4pi+Y5g2qHUTxGRwYQqKZJUnacJkLf8A13QMHh7VAWwaXof0aC8Jm+WVgONfIMIcHZaH2OFFB3GjWiOAMKjIbxaChMMJiQf8QVbd4jitZQRzfEpRj43okD3U1+zRMcUJtd0yFaCi5yzNwk1Yipg1+U8vzICZLkZJcyBQMgkBOEV8l6YEKgaFNE29JAmnuJ13gTK1pmtZBuLlKZ/XBblWKeLoDLyMYMPJT0lQQBwuy7JBpVQmZx2eYZ8nzMoKltL1Rn1Zt2Uncilm8GzG1qKYSGFHsSltEhaF5eKXBcwipKxMNBXI3CQr43Zl3JOEO1crjH0qIYjj+Yd4VOe0bOnDEHCxZzvWJNMBUObq7EKtFRkUDro2a2Z+PmRZcVaMpLkre5xl2LzRxHfpUo8ssRLIEL5UDJBlgKhAShVY9Ono24bVHcdlMfZigwcyD8k3eUBSeZ1mqQUsRLDYHRjs7iAxQBwPmnIt9wjadAtPfpXg+OsjsXS55gU4iRk61pWXuKn8iajMKjokYSgxcGOMohASDO7cFJUhBCw828TMrH4gQla9lUBuU2WhqkEE7Dj7J4u5Bi29p+YQCYyD1ckgIBGX7geqWSrIGWvwV1kx3s1YA0VYUGy1pdt0cfkTxZ0X0xpP4FZoqZFHk1GeimVpU/K4ODlWEpwc0tLRh+YUKygy5+MUQtjkLfkvgknMxZK87maNBjvUApAWg+Z1LJ+DUdvbguM0Tx9aHOM6XjhJSzlGLLT3aVo29BzDPUWm2A375Zx317NW7ajvJ0w5MyHTNsJjWZp7J7NoShtsnKzTEGE8Wcs9jTnaSAOA5JRsxbyJ03pNkjqaFsicHRelLOVB6axnTpX1KMBoIVFDzHRgKAMn5yh8S8qnCiBwj6nnsqfNENk2TKUhPJLehkegahHvCLyf4R4dh+JOHUDwLw2S+L2ee0ZbyXFJOMLY+Yj5kAlIsBqZMaiVgUr8WodRAFLnJK6WUed+jkBVBFOE5MhHbmTiZWg6ZgYK2BkGNyUtZgYR2BhN8yYIrkithGRBQZ4TkISiCGijYxzHDlCJY8tcxjTAhI8K2LRBy7Fwm2Eo6o04Rh7Ecdu19dJtlJLbRBk4PwwUhEOWR7DPQyylvZfYKwzgBgcO/d2DYwxrGeM0U2+RdKJ3IjEt8+xiFHGDjqdMTwOrDkcisZSGTm4wzSu+FsDlKzLjTjbYOjxOiyhWFsM8BcLQKxzkpPMnVjhbQbCPHsx5FpKzbETC4jYnE9TGBVcowiRhfm8f/L0blgbt1SsicsLTxH7mMqZScWVyrfFoa5P4u5oysRQDzYECxZFChZPsXBqIYQ5xDGGas/IyFEitAmW8Nk7RwIaKfUG4oIqiVYsOZZEIUHggCSPeBxzjqKw2U0OCWxdHKGAYsd+UJezRiJtUQiNk1hCgjMfEYx5QXci9B6eeDlb7vE6sbQosiKK2zPKgMMJVmjnC9FUL8/ZN77giphGYIUrI23fi6YcrwMJii8g9OCxJx53EWmWIqVLWaJITlsrmvYvgmSlrmfMRzRYOVpO5K25JlyDFpMcdkuiDmVgomsHOCtyaWj+LM2RY5sXhJaY+KoHYtp5xgs2TcwJnRwRtsU5O/5FRfJ8f/QSKZ+xwiGIgxU1SYzNnSmTIssjc6Q3iVzWk9oolc0vDuMWCt4ENi7Mky5KAhhi1EpWMcnQSpSxaa8NJLaOyYSPlZJ17xhSKhdD4z4i10xnkfDBCUPMW2iT2BKZMp4gRSVtsHU4bJHDnADD05ErkwxNk/DueKABfNQ/T3CgbsLocQAA3TQGAjCsB8H4UQTAADKEAABmGBkiaFgCUTQ0RojAAMAAawABRIYsGEMARRgAAE84NMAACawc0AASisJHfwTAMBRFiCIRjMAEBMASKR/wrHSOkao0wAAekxlj7GABkosSisZAwuUI4RhMaYiLxmAAQeMxBgPxwTIRZOSc0wAPgs0wRQqn1DgbcEQVo2mZPKZzu5koanPCAfs7kUgJAOMOdcBRlzCRVioB5hCDsXnjR0YY8xnjWm4uCAALKaAQAANXwzgGAMXbiRxg1AJgkhJAwCy9EHLWmUOlYvGwUY4Rinmeo3JnjdmgsuEK8VjDWGcMwDwwRojYBiNVe69h3D+HCMkaa0UBLbG8t+c6qRgAJAAIhQyhhoTAUBsBICAKwJWyvZZgIBlbbX1OsAmMttbo3etMH65Nobe3LCSBu+NgbJGTtne8zcHzVBANAA=)

Reply:
dropping `^ 0xFFFFFFFF` mask fixes it. with backing table, you have computed value, no need for extra work.